File: backend-Java/src/main/java/pl/damkac/facility/FacilityService.java
package pl.damkac.facility;

import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
@RequiredArgsConstructor
class FacilityService {

    private final FacilityRepository facilityRepository;

    public List<Facility> getAll(){
        return  facilityRepository.findAll();
    }
}

File: backend-C#/DoctorsOffice/Facilities/FacilityService.cs
using System.Collections.Generic;
using System.Linq;

namespace DoctorsOffice.Facilities
{
    public class FacilityService
    {
        private readonly FacilityRepository _facilityRepository;

        public FacilityService(FacilityRepository facilityRepository)
        {
            _facilityRepository = facilityRepository;
        }

        public List<Facility> GetAll()
        {
            return _facilityRepository.FindAll().ToList();
        }
    }
}